### DISCUSSION TITLE
Insights Panel: Value

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#10640] [Feature Requests]
We need a new (simple) panel type for showing a field value. Options:

* Collection
* Field (any)
* Sort Field
* Sort Direction
* Filters
* Prefix
* Suffix
* Format Value
* Color
* Conditional Formatting

The main difference between this and `metric` is that metric runs aggregation, so it only works with numbers. However, this `value` panel will work with strings too.

**Comment by azrikahar** [comment]
I believe this has been implemented by #10828 👍 (which makes metrics accept strings as well rather than making a separate panel)
